src/main/webapp/app/shared/markdown-editor/commands/orderedListCommand.ts

```diff
@@ -11,24 +11,23 @@ export class OrderedListCommand extends Command {
      * @desc Use the Markdown language for creating/removing an ordered list
      */
     execute(): void {
-        const selectedText = this.getSelectedText();
-        this.splitText(selectedText);
+        const extendedText = this.getExtendedSelectedText();
+        this.splitText(extendedText);
     }
 
     /**
      * Splits the text and performs the necessary manipulations.
      * @function splitText
```

Comment:
```suggestion
     *
```
I don’t think this is needed as part of the JSdoc any more.